src/go/configgenerator/listener_generator.go

```diff
@@ -319,15 +319,21 @@ func makeGrpcStatsFilter() *hcmpb.HttpFilter {
 		EmitFilterState: true,
 		PerMethodStatSpecifier: &gspb.FilterConfig_StatsForAllMethods{
 			StatsForAllMethods: &wrappers.BoolValue{
-				Value: true,
+				// Right now, StatsForAllMethods is set to true by default for backward
```

Comment:
comment is confusing.   can we remove it?
or can we just remove the whole StatsForAllMethods?  what is the default it is removed?

Reply:
b/157791398, default is StatsForAllMethods=true now and will be StatsForAllMethods=false in the future.

Reply:
ok, in that case, we set it to false.   set to true will use more memory if number of methods is large